### ISSUE TITLE
[Bug]: tests with `--parallel` flag fails starting from PEST 2.29.1

### ISSUE THREAD

**Issue by eleftrik** [pestphp/pest#1053]
### What Happened

I have a Laravel `10.39.0` application with ~1000 tests.
Until PEST `2.29.0`, everything runs smoothly: tests are always passing.

Starting from PEST **2.29.1** (even with 2.30.0 version), I got a strange behavior:

- tests are passing when running without `--parallel` flag
- tests are NOT passing when running with `--parallel` flag and I got this error:

```
In WorkerCrashedException.php line 41:

  The test "PARATEST='1' TEST_TOKEN='4' UNIQUE_TEST_TOKEN='4_6595627841337' /Users/..../myproject/tests/Feature/SomeFolder/SomeTest.php" failed.

  Exit Code: 0(OK)

  Working directory: /Users/....../myproject

  Output:
  ================


  Error Output:
  ================

```

(no other details are provided)

If I mark every test inside _SomeTest.php_ file with `->todo();`, such an error still occurs.
If I delete _SomeTest.php_, such an error still occurs, on the next test file inside _SomeFolder_.

For the moment, I've fixed version `2.29.0` in my composer.json to bypass the error.

Please let me know if I can provide you with other details.
Thank you

### How to Reproduce

`XDEBUG_MODE=off ./vendor/bin/pest --parallel --bail --colors=always` to run the whole suite

### Pest Version

2.29.1

### PHP Version

8.2.14

### Operation System

macOS

**Comment by Rydgel** [comment]
I have the same problem.
With the same PHP version as well.
Same behaviors, the tests in itself runs fine, and if I remove the test suite where this error occurs, it moves to the next file.

**Comment by eleftrik** [comment]
@Rydgel Did you try to downgrade PEST to `2.29.0` version? Up now, this is the only temporary fix I have been able to find.

**Comment by Rydgel** [comment]
@eleftrik I did downgrade and it fixed my issue 👍

**Comment by eleftrik** [comment]
@Rydgel I'm glad it worked.

I would like to understand why this error is happening, but I'm missing some debugging skills related to PHPUnit / PEST.
Is there anyone who can help me? Thanks.

**Comment by Rydgel** [comment]
It seems related to PHPUnit update

https://github.com/pestphp/pest/compare/v2.29.0...v2.29.1

**Comment by calebdw** [comment]
This has happened to me as well *but only with xdebug enabled*, no failures happen when removing the `zend_extension=xdebug.so` from the ini files

Edit: the exit code I am getting is `137` and could be due to an OOM error instead of the same issue the OP is experiencing

**Comment by eleftrik** [comment]
Currently, I'm using Pest `2.32.2` and this error has disappeared.

**Comment by devajmeireles** [comment]
Thanks for your feedback, @eleftrik ! 

Make sure to upgrade your PestPHP to the latest version to fix this issue.

**Comment by Rydgel** [comment]
@eleftrik it was fixed by https://github.com/sebastianbergmann/phpunit/issues/5662

**Comment by eleftrik** [comment]
> @eleftrik it was fixed by [sebastianbergmann/phpunit#5662](https://github.com/sebastianbergmann/phpunit/issues/5662)

Thanks!
